cloud-etl/rds-statements.sql

```diff
@@ -0,0 +1,3 @@
+CREATE STREAM eventlogs (eventSourceIP VARCHAR, eventAction VARCHAR, result VARCHAR, eventDuration BIGINT) WITH (KAFKA_TOPIC='rds-eventlogs', VALUE_FORMAT='JSON');
```

Comment:
Trying to piece together why a different topic name is needed for the RDS case but I'm not seeing it. Was something broken?

Reply:
It's needed because the RDS connector will fail if you provide a `topic.prefix` of "".  As a result, when the connector creates a sink topic of `prefix-TableName`, the base ksqlDB stream needs to have the topic with that name.